### PR TITLE
docs: Allow viewing `next` docs in version switch

### PR DESCRIPTION
**Pull request by iRoachie**
Allows to view the latest changes in docs.

screen shot 2018-02-28 at 7 08 20 am

### CODE REVIEW

File: website/pages/en/versions.js [hunk 1]
```diff
@@ -15,6 +15,45 @@ const CWD = process.cwd();
 const siteConfig = require(CWD + '/siteConfig.js');
 const versions = require(CWD + '/versions.json');
 
+class VersionItem extends React.Component {
+  render() {
+    const version = this.props.version;
+    const versionName = version === 'next' ? 'Master' : version;
```
Comment (xavier-villelegier) [inline website/pages/en/versions.js]: Did you use the "Master" name for simplicity or is it a typo ? 😄

Comment (iRoachie) [reply website/pages/en/versions.js]: Yea since they’re not sssociated to any version you need to explicitly set a name.

Comment (xavier-villelegier) [reply website/pages/en/versions.js]: I meant "Master" instead of "Next", my bad*

Comment (iRoachie) [reply website/pages/en/versions.js]: Lol what should we name it? I just took the pattern from the react-native docs

Comment (Monte9) [reply website/pages/en/versions.js]: @iRoachie I think it's better to name it `next` for consistency sake.

## Header version name
screen shot 2018-03-07 at 12 13 13 am

##Display version name
screen shot 2018-03-07 at 12 13 18 am

This is kinda confusing. Thoughts?

Comment (iRoachie) [reply website/pages/en/versions.js]: Changed in https://github.com/react-native-training/react-native-elements/pull/915/commits/f319ad3f2a93d7f2ff9107d0027acfbdaf39e04f

**Pull request merged**